commerce-account-admin-web/src/main/java/com/liferay/commerce/account/admin/web/internal/display/context/CommerceAccountAdminDisplayContext.java

```diff
@@ -111,11 +111,8 @@ protected Boolean getActive() {
 		if (navigation.equals("active")) {
 			return true;
 		}
-		else if (navigation.equals("inactive")) {
-			return false;
-		}
 
-		return null;
+		return false;
```

Comment:
@alecsloan you can change the signature of the method than to bool and use primitive